hw/bsp/samd21/boards/metro_m0_express/metro_m0_express.ld

```diff
@@ -40,7 +40,7 @@ MEMORY
 }
 
 /* The stack size used by the application. NOTE: you need to adjust according to your application. */
-STACK_SIZE = DEFINED(STACK_SIZE) ? STACK_SIZE : DEFINED(__stack_size__) ? __stack_size__ : 0x2000;
+STACK_SIZE = DEFINED(__stack_size__) ? __stack_size__ : 0x2000;
```

Comment:
clang linkder does not seem to support self-check defined for symbol, if not change, result is STACK_SIZE = 0